oar/cli/cmd_check_blocking_sec_alerts.py

```
@@ -0,0 +1,87 @@
+import logging
+
+import click
+
+from oar.core.advisory import AdvisoryManager
+from oar.core.const import *
```

Ruff (0.15.21)

[error] `from oar.core.const import *` used; unable to detect undefined names

(F403)